io/post_fv3.F90

```diff
@@ -679,6 +695,12 @@ subroutine set_postvars_fv3(wrt_int_state,grid_id,mype,mpicomp)
       dtq2 = wrt_int_state%dtp
       nphs = 2.
       dt   = dtq2/nphs
+
+      allocate(extsmoke(ista:iend,jsta:jend,lm))
```

Comment:
Do we need to allocate those array for all the applications?

Reply:
Right now they are for RRFS only.

Reply:
This may increase the memory requirement, I want to confirm for other applications these arrays will not be allocated? I do not see an if statement to limit the allocation just for RRFS.